snippets/change-streams-monitor/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@edmallia/snippet-change-stream-monitor",
+  "snippetName": "change-stream-monitor",
+  "version": "0.0.12",
+  "description": "Mongosh snippet that allows users to monitor Change Streams on the current server.",
+  "main": "index.js",
+  "license": "SSPL",
```

```suggestion
  "license": "MIT",
```

(or change the LICENSE file, because that one currently contains a LICENSE file, but I think MIT is a good call here)

---

Good catch, thanks. I don't have a preference so feel free to change the license as required.